### PR TITLE
Rename SwiftGRPC to GRPC

### PR DESCRIPTION
**Pull request by glbrntt**
- The Swift package was renamed from SwiftGRPC to GRPC
- The Swift module for the NIO variant was renamed from SwiftGRPCNIO to GRPC
- Classes whose names contained NIO had the NIO part removed
- The Echo example was moved from EchoNIO to Echo
- The Google Natural Language example was moved from NaturalLanguageNIO to NaturalLanguage (NaturalLanguage wasn't removed when the CgRPC code was deleted to it was deleted here, which is why there _appear_ to be code changes)
- The .xcodeproj was renamed from SwiftGRPC.xcodeproj to GRPC.xcodeproj

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,9 +169,9 @@ testing with the following versions:
 - Swift 4.0
 - swift-protobuf 1.3.1
 
-## `SwiftGRPCNIO` package
+## `GRPC` package
 
-`SwiftGRPCNIO` is a clean-room implementation of the gRPC protocol on top of the [`SwiftNIO`](http://github.com/apple/swift-nio) library. This implementation is not yet production-ready as it lacks several things recommended for production use:
+`GRPC` is a clean-room implementation of the gRPC protocol on top of the [`SwiftNIO`](http://github.com/apple/swift-nio) library. This implementation is not yet production-ready as it lacks several things recommended for production use:
 
 - Better test coverage
```
Comment (MrMage) [inline README.md]: Mind updating these points? Most of them are outdated, and we might even want to remove the "not yet production-ready" part.

Comment (MrMage) [inline README.md]: Also, there is `grpc-swift` mentioned below, under "license". Replace with "this package"?

Comment (glbrntt) [reply README.md]: Sure; I'll also do a complete update of the README soon!

Comment (glbrntt) [reply README.md]: Done!

**Comment by glbrntt** [comment]
Looks like `testBidirectionalStreamingTimeoutAfterSending` isn't happy on Linux; I'll look into this.

**Pull request merged**